staging/src/k8s.io/sample-controller/artifacts/examples/crd-status-subresource.yaml

```diff
@@ -1,13 +1,35 @@
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   name: foos.samplecontroller.k8s.io
+  annotations:
+    "api-approved.kubernetes.io": "unapproved, experimental-only"
```

Comment:
perhaps append "; please get an approval from Kubernetes API reviewers if you're trying to develop a CRD in the *.k8s.io or *.kubernetes.io groups" to the annotation, and add a comment that links to https://github.com/kubernetes/enhancements/blob/master/keps/sig-api-machinery/2337-k8s.io-group-protection/README.md

so people who read this example and try to develop a CRD will know the next steps.